src/components/App/App.js

```diff
@@ -1 +1,21 @@
+import React, { Component } from 'react'
+import AllMovies from '../AllMovies/AllMovies'
 
+import movieData from '../../movieData'
+
+class App extends Component {
+  constructor() {
+    super()
+    this.state = {
+      movies: movieData.movies
+    }
+  }
+
+  render() {
+    return (
+      <AllMovies movieData={this.state.movies}/>
```

Comment:
I'm excited to see this state being passed down to AllMovies! The class component structure is really coming together. I'm unclear on whether we should store the state for all of the movie in App or maybe in AllMovies. We should ask your mentor when we meet